.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: android
 
-groovy:
```

Comment:
Well, I'll be damned.  I no longer remember why I thought this would create a build matrix in Travis, but I can't find any trace in their github that it ever worked.

Too bad - 2.3.9 introduced a breaking change of some sort that we worked around.  It's less and less relevant as time goes by, but it would be nice to have test coverage there.